script.js

```diff
@@ -0,0 +1,4 @@
+function test(x,y)
+let result = x + y;
```

Comment:
algorithm. boolean, strings

Reply:
ok thanks for the heads up

Reply:
hey i changed something do you like it, if so approve and lets merge